src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String EMPTY_STRING = "";
+    private static final String VALID_PASSWORD = "abcdef";
+    private static final String NOT_VALID_PASSWORD = "abcde";
+    private static final String VALID_LOGIN = "fedcba";
+    private static final String NOT_VALID_LOGIN = "fedcb";
+    private static final User EXISTING_USER = new User(VALID_LOGIN, "password", 20);
+    private static final Integer VALID_AGE = 18;
+    private static final Integer NOT_VALID_AGE = 17;
+    private static final int MIN_FIELD_LENGTH = 6;
+
+    private RegistrationServiceImpl registrationService;
```

Comment:
```suggestion
    private RegistrationService registrationService;
```
you abstract type for variable type